### PR TITLE
privacy policy page

### PR DESCRIPTION
**Pull request by jnakaso**
## What

> Created privacy-page
> Promoted and reused a Masthead component
> Content 

## Why Do

> Completeness
> Masthead allows polishing to affect multiple pages

## How Do

> Could use additional styling input

## Show Me

privacy-page

### CODE REVIEW

File: components/Masthead.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/*
+* Masthead.tsx
+* @2023 Open Seattle
+*/
+
+import { Stack, Typography, styled, useMediaQuery, useTheme } from "@mui/material"
+
+type MastheadProps = {
```
Comment (santina) [inline components/Masthead.tsx:8]: This is nice. We should consider using this whenever we have title that does not have image then!

File: pages/privacy.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+import { ReactNode } from 'react'
+
+type CardCopyTextProps = {
+    title?: string,
+    children: ReactNode
+}
+const CardCopyText = ({ title, children }: CardCopyTextProps) => {
+    const isSmallScreen = useMediaQuery('(max-width:600px)')
+    return (
+        <>
```
Comment (santina) [inline pages/privacy.tsx:26]: nit: you don't need ```<> ..</>```

File: pages/privacy.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+import Masthead from 'components/Masthead'
+import { withBasicLayout } from 'components/layouts'
+import { ReactNode } from 'react'
+
+type CardCopyTextProps = {
+    title?: string,
+    children: ReactNode
+}
+const CardCopyText = ({ title, children }: CardCopyTextProps) => {
+    const isSmallScreen = useMediaQuery('(max-width:600px)')
```
Comment (santina) [inline pages/privacy.tsx:24]: We cannot use breakpoint here?

Comment (santina) [reply pages/privacy.tsx:24]: `const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'))`

File: pages/privacy.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+                    </CardCopyText>
+
+                    <CardCopyText title="Changes to This Privacy Policy" >
+                        <p>We may update our Privacy Policy from time to time. Thus, we advise you to review this page periodically for any changes. We will notify you of any changes by posting the new Privacy Policy on this page. These changes are effective immediately, after they are posted on this page.</p>
+                        <p>Our Privacy Policy was created with the help of the TermsFeed Privacy Policy Template.</p>
+                    </CardCopyText>
+
+                    <CardCopyText title="Contact Us" >
+                        <p>If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us at privacy@openseattle.org</p>
```
Comment (santina) [inline pages/privacy.tsx:133]: is this email valid?

File: pages/privacy.tsx [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+                </Typography>
+            </Box>
+        </>);
+}
+
+type ListItemProps = {
+    text: string
+}
+const ListItem = ({ text }: ListItemProps) => {
+    const isSmallScreen = useMediaQuery('(max-width:600px)')
```
Comment (santina) [inline pages/privacy.tsx:46]: same comment as above

**Review by santina — APPROVED** [review]
Some minor comments. Overall looks good. Thank you!

File: components/Masthead.tsx (hunk 1 above)
Comment (jnakaso) [reply components/Masthead.tsx:8]: I'll do that after this is merged into rebrand-2023

File: pages/privacy.tsx (hunk 2 above)
Comment (jnakaso) [reply pages/privacy.tsx:26]: whoops

File: pages/privacy.tsx (hunk 3 above)
Comment (jnakaso) [reply pages/privacy.tsx:24]: Likewise, i'll fix after merge.

File: pages/privacy.tsx (hunk 4 above)
Comment (jnakaso) [reply pages/privacy.tsx:133]: good question.

**Pull request merged**